### PR TITLE
[lint] Remove unused variables from CircleGraph and PartEditor

### PR DESCRIPTION
**Pull request by dayo09**
This commit removes unused variables and parameters from CircleGraph and PartEditor.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

----

From #1261 
For #1253

### CODE REVIEW

**Review by llFreetimell — APPROVED** [review]
LGTM

File: src/CircleGraph/CircleViewer.ts [hunk 1]
```diff
@@ -123,9 +123,7 @@ export class CircleViewerProvider implements
   }
 
   // CustomReadonlyEditorProvider implements
-  async openCustomDocument(
-      uri: vscode.Uri, openContext: {backupId?: string},
-      _token: vscode.CancellationToken): Promise<CircleViewerDocument> {
+  async openCustomDocument(uri: vscode.Uri): Promise<CircleViewerDocument> {
```
Comment (dayo09) [inline src/CircleGraph/CircleViewer.ts:126]: @seanshpark 

I understand that some of those parameters are for later usage. 
I suggested removing those unused-variables, but if you would like to leave them, let me know.
I will change this PR to put an underscore(_) to those unused parameters to let people know that those are not used and be ignored by the 'no-unused-variables' rule.

Comment (dayo09) [inline src/CircleGraph/CircleViewer.ts:126]:
```suggestion
  async openCustomDocument(uri: vscode.Uri, _openContext: {backupId?: string},
      _token: vscode.CancellationToken): Promise<CircleViewerDocument> {
```
Like this.

(seems you already put the underscore for _token..?)

File: src/CfgEditor/CfgEditorPanel.ts [hunk 2]
```diff
@@ -72,8 +72,7 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
   constructor(private readonly context: vscode.ExtensionContext) {}
 
   public async resolveCustomTextEditor(
-      document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel,
-      _token: vscode.CancellationToken): Promise<void> {
+      document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel): Promise<void> {
```
Comment (seanshpark) [inline src/CfgEditor/CfgEditorPanel.ts:75]: Please don't remove _token if possible

File: src/CircleGraph/CircleViewer.ts (hunk 1 above)
Comment (seanshpark) [reply src/CircleGraph/CircleViewer.ts:126]: I didn't know about the underscore and the tool you're going to use.
But please don't remove _token if possible.

File: src/CircleGraph/CircleViewer.ts [hunk 3]
```diff
@@ -137,9 +135,8 @@ export class CircleViewerProvider implements
   }
 
   // CustomReadonlyEditorProvider implements
-  async resolveCustomEditor(
-      document: CircleViewerDocument, webviewPanel: vscode.WebviewPanel,
-      _token: vscode.CancellationToken): Promise<void> {
+  async resolveCustomEditor(document: CircleViewerDocument, webviewPanel: vscode.WebviewPanel):
```
Comment (seanshpark) [inline src/CircleGraph/CircleViewer.ts:138]: Please don't remove _token if possible

File: src/PartEditor/PartEditor.ts [hunk 4]
```diff
@@ -352,7 +352,7 @@ class PartEditor implements PartGraphEvent {
   }
 
   // PartGraphEvent implements
-  public onSelection(names: string[], tensors: string[]) {
+  public onSelection(names: string[]) {
```
Comment (seanshpark) [inline src/PartEditor/PartEditor.ts:355]: Please don't remove tensors, this is interface implementation.

File: src/PartEditor/PartEditor.ts [hunk 5]
```diff
@@ -387,8 +387,7 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider, Part
   }
 
   public async resolveCustomTextEditor(
-      document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel,
-      token: vscode.CancellationToken): Promise<void> {
+      document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel): Promise<void> {
```
Comment (seanshpark) [inline src/PartEditor/PartEditor.ts:390]: please change token to _token

File: src/CircleGraph/CircleViewer.ts (hunk 1 above)
Comment (dayo09) [reply src/CircleGraph/CircleViewer.ts:126]: Sure :-D

**Comment by dayo09** [comment]
@seanshpark @llFreetimell PTAL :-D

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**